src/AppBundle/Controller/DefaultController.php

```diff
@@ -37,12 +46,28 @@ public function indexAction(Request $request)
 
     /**
      * @Route("/registration", name="app_identity_registration")
+     *
+     * @throws \InvalidArgumentException
```

Comment:
The method isn't throwing this exception in it's method body. Consider removing this annotation.